create-ssirius-standalone/template/src/componentMap.ts

```diff
@@ -103,7 +106,46 @@ const component_map = {
                 errorHandler: FPErrorHandler
             })
         }
+    },
+    fpRegistration: {
+        component: FingerprintRegistration,
+        props: {
+            getFingerprint: async function() {
+                const {data} = await axios.get('http://localhost:9907/EKYC/Fingerprint');
+                return data.ImageBase64;
+            },
+            register: async function(fingerprints: FingerprintMap, dependencyData: any) {
```

Comment:
@jeffk-kiva If you could just take a look at this function and let me know if this is how the FP registration is supposed to work, that would be wonderful. I attempted to model it off of https://github.com/kiva/protocol-integration-tests/blob/main/test/v2.remote.employee.guardian.fingerprint.e2e-spec.ts#L91

Reply:
I think this looks right!